manager/manager/typeresolver.go

```diff
@@ -92,18 +93,27 @@ func (tr *typeResolver) ResolveTypes(config *common.Configuration, imports []*co
 	}
 
 	fetched := map[string][]*common.ImportFile{}
-	toFetch := make([]string, 0, tr.maxUrls)
+	// TODO(vaikas): Need to account for multiple URLs being fetched for a given type.
+	toFetch := make([]*fetchUnit, 0, tr.maxUrls)
```

Comment:
may as well just ditch the maxUrls param here. Why do we even want to limit this?